sn_networking/src/lib.rs

```diff
@@ -385,6 +385,13 @@ impl Network {
             let res = self
                 .put_record_once(record.clone(), verify_store.clone())
                 .await;
+
```

Comment:
Also this is performed only if `verify_store.is_some()`, so maybe add a comment stating that or maybe add in `if verify_storel.is_some() {}` to indicate it?